bootstrap-scripts/raspberry-pi-setup.sh

```diff
@@ -62,4 +62,6 @@ echo Install node bindings for cmake
 sudo npm install -g cmake-js
 
 # Configure abigail-device service
-sudo cp $mydir/config-files/abigail-device.service /lib/systemd/system
+# Note: the ugly bash expansion below replaces all `/` with `\/` so that sed is
+# happy.
+sudo sed "s/<SOURCEDIR>/${source_dir////\\/}/g" $source_dir/config-files/abigail-device.service.in > /lib/systemd/system/abigail-device.service
```

Comment:
so there is 2 things here:

1. bash itself replaces `${source_dir////\\/}` because we used double quotes for the string.
   The basic syntax is `${VARNAME/pattern/replacement}`; when the pattern starts with a `/` it does a global replace ("replace all"). So: the first `/` is the syntax separator, the second `/` is the "global". The third `/` is the actual pattern. The fourth `/` is the syntax separator. And we replace with `\/` where `\` needs to be escaped.
The goal of this is to replace the `/` by `\/` in the path so that sed is happy.

2. sed replaces using `s/pattern/replacement/g`; g means 'global' means 'replace all' :)